recipes/fortran-binary/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "fortran-binary" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: be613276fb3141b81933c508990b8fa5e71f1df04a7423567d2aa78a16267241
+
+build:
+  number: 1
```

This should start from 0
```suggestion
  number: 0
```